tests/x509/test_x509.py

```diff
@@ -4690,7 +4690,7 @@ def test_distinugished_name(self):
         # Nonstandard attribute OID
         na = x509.NameAttribute(NameOID.EMAIL_ADDRESS, "somebody@example.com")
         assert (
-            na.rfc4514_string() == "1.2.840.113549.1.9.1=somebody@example.com"
+            na.rfc4514_string() == "E=somebody@example.com"
```

Comment:
The goal of this test is to see what happens with an OID not in that dict, so we need to use a different OID here, not change the expectations

Reply:
Thank you for the explanation. Hopefully this works better.